### ISSUE TITLE
[Accessibility] ToolStripComboBox/ToolStripTextBox items' name property cannot be null

### ISSUE THREAD

**Issue by Ashley-Li** [dotnet/winforms#3663]
* .NET Core Version: .Net SDK 5.0.100-rc.1.20378.13

* Have you experienced this same bug with .NET Framework?: 
Yes

* More info
This issue can reproduce on contextMenuStrip/MenuStrip.
**Minimal repro::**
1. Extract “[testApp.zip](https://github.com/dotnet/winforms/files/4994154/testApp.zip)” and open it in VS.
2. Build and run the project.
3. Open Accessibility Insights For Windows tool, select focus on form and view result.

**Actual behavior::**
 Use the tool to test out the following error,toolStripComboBox/ToolStripTextBox items' name property cannot be null.
![ComboBox](https://user-images.githubusercontent.com/56664234/88787008-cdd0d800-d1c5-11ea-9802-132379af3292.PNG)

**Expected behavior:**
ToolStripComboBox/ToolStripTextBox items' name property cannot be null.
